packages/files/file_info.pony

```diff
@@ -15,21 +9,79 @@ class val FileInfo
   let filepath: FilePath
 
   let mode: FileMode val = recover FileMode end
+    """UNIX-style file mode"""
```

Comment:
Can we make this more consistent with some of the other field docstrings and add punctuation at the end?